### DISCUSSION TITLE
`renovate-approve` bot setup

### DISCUSSION THREAD

**Discussion by djmitche** [renovatebot/renovate#8201] [Request Help]
I just added the renovate-approve app to my org (all repos).  However, I don't seem to be able to add it as a reviewer on an [open PR](https://github.com/taskcluster/taskcluster/pull/4219) - it doesn't autocomplete in the list of collaborators.  I tried to add access for the account to the repo, but
![image](https://user-images.githubusercontent.com/28673/103795342-dc90a600-5013-11eb-9073-05406795554d.png)

Also, I see
> Important note: Due to a GitHub limitation, it is not possible to assign any app like this one as a CODEOWNER, so unfortunately this bot won't work that way if you have CODEOWNERS set up.

in the app information, but this leaves me hanging: what *can* I do?  Must I stop using CODEOWNERS at all?  Can I configure renovate to add `renovate-approve` as a reviewer on its PRs?

Basically I'm at a dead end and not sure how to proceed.

**Comment by djmitche** [comment]
Ah, I see now that the bot has approved a new PR, so perhaps the inability to add the bot as a reviewer doesn't matter.  I assume it's branch rules preventing that from actually merging.

**Comment by djmitche** [comment]
Huh, but https://github.com/taskcluster/taskcluster/pull/4222 and https://github.com/taskcluster/taskcluster/pull/4223 have not gotten approval.  Is there some kind of intermittency at play here, or are those different somehow from https://github.com/taskcluster/taskcluster/pull/4221?

**Reply by viceice** [reply]
The renovate-approve app only approves auto merge pr's

**Reply by djmitche** [reply]
those are auto-merge PRs, but I see now that renovate-bot is not the only committer (we have to follow-up on some renovate PRs since it doesn't fully update yarn.lock..).

**Reply by djmitche** [reply]
Oops, you're right, those are modifying different package.json files

**Reply by rarkins** [reply]
It responds to (a) new PRs, and (b) ones with 
![image](https://user-images.githubusercontent.com/6311784/103903379-1e951700-50fc-11eb-9c14-6c3c1c33e24c.png)

It holds back reviews from others because there's no point adding noise to the PR

**Reply by djmitche** [reply]
OK, I think all the bits were in place this time:

https://github.com/taskcluster/taskcluster/pull/4229

![image](https://user-images.githubusercontent.com/28673/103960550-64fc6d00-5120-11eb-9806-09083745bd86.png)
![image](https://user-images.githubusercontent.com/28673/103960562-6d54a800-5120-11eb-9577-0c0d38ffcfab.png)
![image](https://user-images.githubusercontent.com/28673/103960574-76457980-5120-11eb-86f8-5c8d3bc1c61d.png)

There is only one commit. by renovate.

But no automerge...

**Reply by viceice** [reply]
That pr is now merged by renovate. 😏

**Reply by djmitche** [reply]
Indeed, but an hour and 15 minutes after the approval.  Is that delay configurable?

**Reply by viceice** [reply]
nope, if you don't manually trigger renovate via dependency dashboard, it will run hourly by default
